contracts/cw1/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+#![allow(dead_code)]
+use cosmwasm_std::{Deps, DepsMut, Env, MessageInfo, Response, StdError};
+
+use sylvia_derive::interface;
+
+#[interface(module=msg)]
+pub trait Cw1 {
+    type Error: From<StdError>;
+
+    #[msg(exec)]
+    fn add_member(
+        &self,
+        ctx: (DepsMut, Env, MessageInfo),
+        member: String,
+    ) -> Result<Response, Self::Error>;
+
+    #[msg(query)]
+    fn find_member(&self, ctx: (Deps, Env), member: String) -> Result<Response, Self::Error>;
+}
+
+#[cfg(test)]
+mod tests {
```

Comment:
I see you are testing if serialization->deserialization is an identity function which is good, but it is a bit missing here. You want to make sure, that the in-between product is proper. In particular, you want to make sure, that deserialization of JSONs you expect is proper.

Testing the serialized JSONs would be a bit strange, as you will need to consider formatting, and so, and maintaining it would be hell. What I would do is deserialize hand-crafted JSON messages (with `cosmwasm_std::from_slice`).